source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Incremental Backup
+category: feature
+authors: nsoffer, derez
+feature_name: Incremental Backup
+feature_modules: imageio,engine
+feature_status: Design
+---
+
+# Incremental Backup
+
+
+## Summary
+
+Incremental backup allows a user to backup only the changed blocks in
+all disks belonging to a VM snapshot. The typical use case is backing
+up the current changes in the active snapshot. Another use case may be
+backing up the previous snapshot if a user added a snapshot since the
+last backup.
```

Comment:
We need to mention that users using RAW files will have to have a implicate snapshot created on top to allow incremental backup.

Reply:
done